clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -21,6 +21,20 @@
    :type       type-name
    :attributes (dissoc value :id)})
 
+(defn- update-vals [map vals f]
+  (reduce #(update-in %1 [%2] f) map vals))
+
+(defn- parse-date [value]
```

Comment:
My thinking was that `core.clj` seems to be growing a bit large, and SRP, and other OO stuff that may or may not apply in this strange FP world. :) Just observating out loud, here, BTW, so don't feel obligated to take any of this under actual consideration.